doc/Interns/elliesekine/MyFirstCommit.md

```diff
@@ -0,0 +1 @@
+5/6/2020   
```

Comment:
Very good.  One last request.  I've "forgotten" to mention.  Please add the time to this file.